### PR TITLE
Expands variety of items in NPC NC itemgroups

### PR DESCRIPTION
**Pull request by chaosvolt**
Adds assorted weapons and clothing to the groups NPCs use to determine their starting gear.

If any of these additions seem kinda weird to the point of needing a change or removal, will alter these as needed.

### CODE REVIEW

File: data/json/npcs/NC_ARSONIST.json [hunk 1]
```diff
@@ -155,14 +164,16 @@
...
+      ["remington_870", 25]
     ]
   },{
     "type" : "item_group",
     "id" : "NC_ARSONIST_smg",
     "items":[
       ["uzi", 50],
-      ["tec9", 50]
+      ["tec9", 50],
+      ["sten", 10],
```
Comment (Rivet-the-Zombie) [inline data/json/npcs/NC_ARSONIST.json]: Need to kill the comma on the end of this line.

File: data/json/npcs/NC_HUNTER.json [hunk 2]
```diff
@@ -71,7 +74,8 @@
       ["browning_blr", 10],
       ["remington_700", 10],
       ["savage_111f", 5],
-      ["marlin_9a", 25]
+      ["marlin_9a", 25],
+      ["m1903", 5],
```
Comment (Rivet-the-Zombie) [inline data/json/npcs/NC_HUNTER.json]: Need to kill the comma on the end of this line.

**Comment by chaosvolt** [comment]
Doh. That's what happens when you make a PR in the middle in the night. @_@

**Comment by chaosvolt** [comment]
There we go. Now, I'm pondering whether these numbers are sensible. Also, should I maybe replace the M1903 additions with Mosin Nagants? They'd likely be the more common WWII surplus rifles.

**Comment by chaosvolt** [comment]
And thank you for the merge.

**Pull request merged**